aries-site/src/examples/templates/dashboards/TwoColumnDashboard.js

```diff
@@ -0,0 +1,100 @@
+import React, { useContext } from 'react';
+import { Box, Grid, ResponsiveContext } from 'grommet';
+import {
+  AppContainer,
+  PageContainer,
+  PageContainerContext,
+} from '../page-layouts/components';
+import { ContentArea } from '../page-layouts/anatomy/components';
+import { PageHeader } from './components';
+import { ServerHealth } from './content';
+
+export const TwoColumnDashboard = () => (
+  <AppContainer background="background-back">
+    <ContentArea title="Global Header" />
+    <PageContainer>
+      <PageHeader title="Dashboard" />
+      <PageContent />
+    </PageContainer>
+    <ContentArea title="Global Footer" />
+  </AppContainer>
+);
+
+const parentGrid = {
+  columns: {
+    xsmall: '100%',
+    small: '100%',
+    medium: ['auto', 'auto'],
+    large: ['auto', 'medium'],
+    xlarge: ['auto', 'medium'],
+  },
+  gap: {
+    xsmall: 'large',
+    small: 'large',
+    medium: 'medium',
+    large: 'large',
+    xlarge: 'large',
+  },
+};
+
+const firstChildGrid = {
+  columns: {
+    xsmall: '100%',
+    small: '100%',
+    medium: '100%',
+    large: ['auto', ['small', 'medium']],
+    xlarge: ['auto', ['small', 'medium']],
+  },
+  gap: 'medium',
+};
+
+const secondChildGrid = {
+  columns: {
+    xsmall: '100%',
+    small: '100%',
+    medium: '100%',
+    large: { count: 'fit', size: ['medium', 'auto'] },
+    xlarge: { count: 'fit', size: ['medium', 'auto'] },
+  },
+  gap: 'medium',
+};
+
+const PageContent = () => {
+  const size = useContext(ResponsiveContext);
+  const { pad } = useContext(PageContainerContext);
+
+  return (
+    <Grid
+      gap={parentGrid.gap[size]}
+      columns={parentGrid.columns[size]}
+      pad={pad}
+    >
+      {/* Content Block 1 is top priority content. At narrow breakpoints, place 
+      as first content element. Otherwise, place in second column. */}
+      {(size === 'small' || size === 'xsmall') && <ContentBlock title="1" />}
+      <Grid gap={firstChildGrid.gap}>
+        <Grid columns={firstChildGrid.columns[size]} gap={firstChildGrid.gap}>
+          <ServerHealth />
+          <ContentBlock
```

Comment:
What is our guidance for cards in the same row with different heights based on their content? Do we want to allow blank space below the card (like content block 3) OR do we want to have cards fill all available space in their row? I am not guiding one way or another but just curious about what has been decided

<img width="1397" alt="Screen Shot 2022-02-02 at 10 56 32 AM" src="https://user-images.githubusercontent.com/12522275/152219989-9c27dc75-a220-48b6-b44d-d468e34fa31a.png">

Reply:
I believe we are leaning toward filling for aesthetic purposes, however there are use-cases where it wouldn't make sense:
- The [activity feed here](https://www.figma.com/file/4rdBkUlHd5MCVq3hvUOXHd/Page-Layouts?node-id=2617%3A22677) doesn't/shouldn't fill
- A DataTable wouldn't fill horizontally (though it probably wouldn't be presented in a card)

If we are actually using Cards, I would think they would fill horizontally and vertically 90+% of the time. Something we should watch out for as we implement more test cases.

Reply:
I think once we feel these cards it will be easier to see how they look but I would also think that they would fill horizontally and vertically 90+% of the time.